src/apprt/gtk/Window.zig

```diff
@@ -14,6 +14,8 @@ const configpkg = @import("../../config.zig");
 const font = @import("../../font/main.zig");
 const input = @import("../../input.zig");
 const CoreSurface = @import("../../Surface.zig");
+const gtk = @import("gtk");
+const gobject = @import("gobject");
```

Comment:
Mitchell's standard order for imports is anything from `std`, 3rd party libs, internal imports.